shared/chat/conversation/messages/account-payment/index.js

```diff
@@ -34,7 +34,11 @@ const AccountPayment = (props: Props) => {
   ) : (
     <React.Fragment>
       <Box2 direction="horizontal" fullWidth={true} style={styles.headingContainer}>
-        <Box2 direction="horizontal" gap="xtiny" style={styles.headingContainer}>
+        <Box2
+          direction="horizontal"
+          gap="xtiny"
+          style={{...styles.headingContainer, marginBottom: globalMargins.xtiny}}
```

Comment:
@chrisnojima Flow checks are complaining about this here, what am I doing wrong?

Reply:
Switch this object out for `collapseStyles([styles.headingContainer, {marginBottom: ...}])`, flow should stop complaining.